class.phpmailer.php

```diff
@@ -840,6 +840,13 @@ protected function addAnAddress($kind, $address, $name = '')
             }
             return false;
         }
+        if ($this->hasMultiBytes($address) && function_exists('idn_to_ascii')) {
```

Comment:
I would add a comment here:

TODO: Write our own "idn_to_ascii" function for PHP 5.2 and earlier.